cli/scripts/cli-tests.js

```diff
@@ -12,7 +12,7 @@ const colors = {
 
 import fs from "fs";
 import path from "path";
-import { execSync, spawn } from "child_process";
+import { spawn } from "child_process";
```

Comment:
This was a unused import – using it as a drive-by fix in this PR to trigger the `/cli` CI pipeline.